src/main/java/edu/kit/ipd/crowdcontrol/objectservice/crowdworking/PaymentJob.java

```diff
@@ -31,7 +31,7 @@ public WorkerRecord getWorkerRecord() {
 
     /**
      * Get the amount to pay
-     * @return
+     * @return the amount to pay in USC
```

Comment:
As far as i googled, the abbreviation is USc, but i wouldn't use it, because people, who are using these abbreviation very often could misunderstand this. I would prefer US cents, US ct or something similar.